src/main/scala/com/gu/newsletterlistcleanse/db/CampaignSentDate.scala

```diff
@@ -0,0 +1,22 @@
+package com.gu.newsletterlistcleanse.db
+
+import java.time.{ ZoneId, ZonedDateTime }
+import java.time.format.DateTimeFormatter
+
+import scalikejdbc.WrappedResultSet
+
+case class CampaignSentDate(
+  campaignId: String,
+  campaignName: String,
+  timestamp: ZonedDateTime
+)
+
+object CampaignSentDate {
+
+  val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS").withZone(ZoneId.of("UTC"))
+
+  def fromRow(rs: WrappedResultSet): CampaignSentDate = CampaignSentDate(
+    campaignId = rs.string("campaign_id"),
+    campaignName = rs.string("campaign_name"),
+    timestamp = ZonedDateTime.from(formatter.parse(rs.string("timestamp"))))
```

Comment:
This can be replaced by `rs.zonedDateTime("timestamp")` if we start using the [version 0.2.4](https://github.com/zaneli/scalikejdbc-athena/releases/tag/0.2.4) as it fixes the bug i encountered

Reply:
Updated to use this suggestion